alma/controllers/admin/AdminAlmaConfig.php

```diff
@@ -32,8 +32,10 @@ public function init()
             'AdminModules',
             true,
             [],
-            ['configure' => 'alma', 'module_name' => 'alma', 'tab_module' => 'payments_gateways']
-        ) . '&configure=alma&module_name=alma&tab_module=payments_gateways';
+            //phpcs:ignore
```

Comment:
can you give a context of which `phpcs:ignore` is ignored here (for example `phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter`)

Reply:
The display is not very well formatted on github, but here are the lines in phpcs:ignore
`//phpcs:ignore`
            `['configure' => $this->module->name, 'module_name' => $this->module->name, 'tab_module' => $this->module->tab]`
        `//phpcs:ignore`
        `) . '&configure=' . $this->module->name . '&module_name=' . $this->module->name . '&tab_module=' . $this->module->tab;`